cmake/onnxruntime.cmake

```diff
@@ -60,9 +60,18 @@ function(download_onnxruntime)
       ${PROJECT_BINARY_DIR}/onnxruntime-win-x64-1.14.0.zip
       /tmp/onnxruntime-win-x64-1.14.0.zip
     )
-    set(onnxruntime_URL  "https://github.com/microsoft/onnxruntime/releases/download/v1.14.0/onnxruntime-win-x64-1.14.0.zip")
-    set(onnxruntime_URL2 "https://huggingface.co/csukuangfj/sherpa-onnx-cmake-deps/resolve/main/onnxruntime-win-x64-1.14.0.zip")
-    set(onnxruntime_HASH "SHA256=300eafef456748cde2743ee08845bd40ff1bab723697ff934eba6d4ce3519620")
+
```

Comment:
Please move lines 57-63 to 64-bit windows
and also
please create a similar version of possible_file_locations
for 32-bit windows.